File: Java/[2 sem] 2DWorldSimulator/code/Grass.java
public class Grass extends Plant {
    private static final int GRASS_STRENGTH = 0;
    private static final int GRASS_INITIATIVE = 0;
    private static final char GRASS_SYMBOL = '*';

    public Grass(int x, int y, World world) {
        super(x, y, world, GRASS_STRENGTH, GRASS_INITIATIVE, GRASS_SYMBOL,0);
    }

    @Override
    public String getSpecieName() {
        return "Grass";
    }
}

File: C(++)/[2 sem] 2DWorldSimulator/code/src/worldObjects/plants/Grass.cpp
//
// Created by szymo on 4/16/2023.
//

#include ".headers/Grass.h"

#define GRASS_STRENGTH 0
#define GRASS_INITIATIVE 0
#define GRASS_SYMBOL '*'

Grass::Grass(int x, int y, World *world) : Plant(x, y, world, GRASS_STRENGTH, GRASS_INITIATIVE, GRASS_SYMBOL) {}

Grass::~Grass() = default;

std::string Grass::getSpecieName() const {
    return "Grass";
}